src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -200,6 +200,8 @@
     public static final String MAX_NUM_INTERVALS_TO_IMPORT_IN_PARALLEL = "max-num-intervals-to-import-in-parallel";
     public static final int INTERVAL_LIST_SIZE_WARNING_THRESHOLD = 100;
 
+    public static final String SHARED_POSIXFS_OPTIMIZATIONS = "genomicsdb-shared-posixfs-optimizations";
```

Comment:
Can you just use `GenomicsDBArgumentCollection.SHARED_POSIXFS_OPTIMIZATIONS` directly in this class instead of duplicating it here?

Reply:
Done.